codec-http/src/main/java/io/netty/handler/codec/http/HttpRequestDecoder.java

```diff
@@ -76,6 +77,34 @@
  */
 public class HttpRequestDecoder extends HttpObjectDecoder {
 
+    private static final AsciiString Host = AsciiString.cached("Host");
```

Comment:
@chrisvest 
I didn't implemented the lowercase cases too, nor *all* header names (or methods), but the ones I have noticed are used the most within a reasonable amount of checks (and disambiguation due to same length collision); but I can implement the lowercases one if you think is worthy (ping some ServiceTalk folk if you can, I have not much experience here to decide :) )

Reply:
And just thinking loud: on Netty 5 this is going to use var handle byte view...yum!
Is so sad I can just use Unsafe to get something similar here (and I still think is not worth, I can re-run the micros to be sure)